cli/dcoscli/package/main.py

```diff
@@ -14,18 +15,24 @@
     dcos package update [--validate]
 
 Options:
-    -h, --help         Show this screen
-    --info             Show a short description of this subcommand
-    --version          Show version
-    --yes              Assume "yes" is the answer to all prompts and run
-                       non-interactively
-    --all              Apply the operation to all matching packages
-    --app              Apply the operation only to the package's application
-    --app-id=<app-id>  The application id
-    --cli              Apply the operation only to the package's CLI
-    --options=<file>   Path to a JSON file containing package installation
-                       options
-    --validate         Validate package content when updating sources
+    -h, --help                          Show this screen
+    --info                              Show a short description of this
+                                        subcommand
+    --version                           Show version
+    --yes                               Assume "yes" is the answer to all
+                                        prompts and run non-interactively
+    --all                               Apply the operation to all matching
+                                        packages
+    --app                               Apply the operation only to the
+                                        package's application
+    --app-id=<app-id>                   The application id
+    --cli                               Apply the operation only to the
+                                        package's CLI
+    --options=<file>                    Path to a JSON file containing package
+                                        installation options
+    --validate                          Validate package content when updating
+                                        sources
+    --package-version=<package_version> Package version to install
```

Comment:
It can be more than 80-chars just want to make sure that it prints "nice" when using `--help`. One option is to move this to a file specified here: https://github.com/mesosphere/dcos-cli/blob/master/cli/setup.py#L81

Thoughts?